packages/react-button/package.json

```diff
@@ -14,26 +14,23 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "just-scripts build",
-    "bundle": "just-scripts bundle",
-    "clean": "just-scripts clean",
-    "code-style": "just-scripts code-style",
-    "just": "just-scripts",
-    "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook",
-    "start:legacy": "just-scripts dev",
-    "start-test": "just-scripts jest-watch",
-    "test": "just-scripts test",
-    "update-api": "just-scripts update-api",
-    "update-snapshots": "just-scripts jest -u"
+    "build": "echo 'TODO: rollup & tsc --declaration && api-extractor'",
```

Comment:
I like Rollup, it's awesome, I would say that it's brilliant for small libraries 💎 

_But,_ I would be careful with Rollup taking in account that Terser on consumer's side often does not perform well. If we will bundle with Rollup we will get a single file with all possible side-effects.

### Real example 1: downshift

I can dig dipper to remember, but functions in this file (https://github.com/downshift-js/downshift/blob/a13308910282c44ea2ebdcbc8a6dfee901df48a2/src/set-a11y-status.js#L26) are producing side-effects that will stay anyway in a bundle.

### Real example 2: non obvious Terser behavior

It's a recent discovery from Fluent's code, I have already shared this case with @dzearing. Constants in [this file](https://github.com/microsoft/fluentui/blob/d2c9430b1470c3d3912a436fcc86d0b6b162a9d5/packages/utilities/src/properties.ts) are well defined, what can go wrong? The first line... 💣 

```ts
const toObjectMap = (...items: (string[] | Record<string, number>)[]) => {};
// will become in JS
var toObjectMap = function () {
    var items = [];
    for (var _i = 0; _i < arguments.length; _i++) {
        items[_i] = arguments[_i];
    }
};
```

Still looks good? But it will not be removed by Terser (_v5.5.1_) ever:

#### Input

```js
var toObjectMap = function () {
    var items = [];
    for (var _i = 0; _i < arguments.length; _i++) {
        items[_i] = arguments[_i];
    }
};

var fooItems = toObjectMap()
```

#### Rollup 2.18.2

```js
!function(){for(var n=[],f=0;f<arguments.length;f++)n[f]=arguments[f]}();
```

#### Webpack 5.19.0

_BTW, try to find a difference 😏_ 

```js
!function(){for(var n=[],f=0;f<arguments.length;f++)n[f]=arguments[f]}();
```

Reply:
Clearly a larger discussion. My comment is resolved as long as nothing in the PR here depends on going one way or the other.

Reply:
@layershifter  interesting. well those are not constants by definitions (as it's dynamic behaviour as you pointed out) - maybe I'm missing something, but that function is needed for runtime, if it would be removed, those constants wouldn't work.

Reply:
discussed the issue with @layershifter offline 💪 . 

In short: There might be an issue within Terser -> property assignments are preventing tree shaking (array indexes, static props) etc